tools/filters/tailWrapper.xml

```diff
@@ -1,10 +1,21 @@
 <tool id="Show tail1" name="Select last" version="1.0.0">
   <description>lines from a dataset</description>
-  <command detect_errors="exit_code">
-perl '$__tool_directory__/tailWrapper.pl' '$input' $lineNum '$out_file1'</command>
+  <command detect_errors="exit_code"><![CDATA[
```

Comment:
Same here, I think we need to bump the version and add the new version to https://github.com/mvdbeek/galaxy/blob/926268ce6e1acd3db4e51ceb72e689899c545b9c/lib/galaxy/tools/__init__.py#L195